index.ts

```diff
@@ -116,11 +116,19 @@ export default class GeoSearch {
       throw new Error('No result')
     }
 
+    const northeastLat = result?.geometry?.viewport.northeast.lat
+    const southwestLat = result?.geometry?.viewport.southwest.lat
+
+    const northeastLng = result?.geometry?.viewport.northeast.lng
+    const southwestLng = result?.geometry?.viewport.southwest.lng
+
+    const comp = southwestLng > northeastLng ? 360 : 0
```

Comment:
I think the naming could be better for this constant. Also I think it would be very helpful with a comment that describes why this is needed